vizro-core/tests/unit/vizro/models/_navigation/test_navigation.py

```diff
@@ -82,7 +82,7 @@ def test_default_nav_selector(self, pages, request):
         built_navigation = navigation.build(active_page_id="Page 1")
         assert_component_equal(
             built_navigation["nav_bar_outer"],
-            html.Div(className="hidden", id="nav_bar_outer"),
+            html.Div(hidden=True, id="nav_bar_outer"),
             keys_to_strip={"children"},
```

Comment:
Does this still pass with `keys_to_strip={}`?

Reply:
Yes, it still passes with `keys_to_strip={}` and `keys_to_strip` not being specified at all.

Reply:
In this case please can you change to `keys_to_strip={}`? Will explain why in a near-future PR